config/config.go

```diff
@@ -164,6 +161,21 @@ type Authorization struct {
 	PolicyErrRetryInterval string `yaml:"policyErrRetryInterval"`
 }
 
+// DebugServer represents the server for debug use.
+type DebugServer struct {
+	// Enable represents if user want to enable debug server functionality.
+	Enable bool `yaml:"enable"`
+
+	// Port represents debug server port.
+	Port int `yaml:"port"`
+
+	// EnableDump represents if user want to enable memory dump functionality.
+	EnableDump bool `yaml:"enable_dump"`
```

Comment:
`EnableDump` => `EnablePolicyDump`?

Reply:
For future if we add more cache then another flag need to be added, so I think enable dump should be better.